WordPress/WordPressShareExtension/ShareExtensionSessionManager.swift

```diff
@@ -94,18 +96,24 @@ import WordPressFlux
             }
```

Comment:
Out of scope here, but I think this `guard let blog...` could be moved to the beginning of the `else` branch because it doesn't depend on the `uploadedMediaCount` value and it does an early return.